org.mixedrealitytoolkit.input/Tests/Runtime/BasicInputTests.cs

```diff
@@ -27,6 +32,9 @@ namespace MixedReality.Toolkit.Input.Tests
     /// </summary>
     public class BasicInputTests : BaseRuntimeInputTests
     {
+        private const string OPENXR_RIGHT_HAND_PREFAB_GUID = "da93d751ddc0f64468dfc02f18d02d00";
+        private const string OPENXR_LEFT_HAND_PREFAB_GUID = "2b468cc4fe6d2b44ebc53b958b38b91a";
```

Comment:
What are these for, they don't appear to be used anywhere?

Reply:
Oh!, remnants of Unity-tests I was using for testing, I forgot to remove them.  Thanks for catching that.  Removed in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/716/commits/d74062c83f752f7bf380e17a3465dce5dd6b7f99 , resolving.